sympy/utilities/tests/test_lambdify.py

```diff
@@ -581,3 +629,71 @@ def test_issue_2790():
 def test_ITE():
     assert lambdify((x, y, z), ITE(x, y, z))(True, 5, 3) == 5
     assert lambdify((x, y, z), ITE(x, y, z))(False, 5, 3) == 3
+
+
+def test_scipy_translation_bessel():
+    if not scipy:
+        skip("scipy not installed.")
+
+    f = lambdify((x, y), besselj(x,y), "scipy")
+    assert f(1, 2) == 0.5767248077568734
+    assert f(3, 4) == 0.43017147387562199
```

Comment:
This is not a good way to test floating point numbers. Please use NumPy's assert_allclose or similar functions.